### DISCUSSION TITLE
GNOME Integration: background app, quick settings

### DISCUSSION THREAD

**Discussion by bennypowers** [input-leap/input-leap#1791] [Feature Requests]
Recent versions of GNOME add two features which are perfectly suited to input leap: Quick Settings (GNOME 43) and Background Apps (GNOME 44). Along with #36, adding support for these GNOME UI features would significantly enhance the input leap user experience for GNOME users.

**Comment by shymega** [comment]
It would, but it would require a lot of effort to work specifically with GNOME. I would rather go for a `libportal`-oriented approach, with a D-Bus interface, so any WM/DE that uses a XDG Desktop Portal, can interface with Input Leap. However, that would be on them to implement the control interface.
